src/dmd/dsymbolsem.d

```diff
@@ -245,6 +245,11 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
         Declaration d = sx.isDeclaration();
         if (d && !d.isTupleDeclaration())
         {
+            // Fix Issue 18429 : if the identifier in the alias this declaration
```

Comment:
Instead of writing "Fix Issue 18429", I suggest just pasting the URL.  It has the issue number in it, and gives the reader of the code immediate access the the issue.

Reply:
Done.